test/app.test.ts

```diff
@@ -400,6 +419,28 @@ test('apps with varying yamlOutputTypes; chart dependencies via custom construct
 
     app.synth();
 
-    expect(fs.readdirSync(app.outdir)).toEqual(testSpec.result);
+    expect(getFilesAndFolders(app.outdir)).toEqual(testSpec.result);
+  }
+});
+
+/**
+ * Get the list of files and folders in the source folder and sub folders (one level deep)
+ * @param sourceDir Folder in which to search for files and folders
+ * main folder
+ */
+function getFilesAndFolders(sourceDir: string) {
+  let result = [];
+  let items = fs.readdirSync(sourceDir);
+  for (let i in items) {
+    // result.push(items[i]);
+    if (fs.lstatSync(path.join(sourceDir, items[i])).isDirectory()) {
```

Comment:
style: We prefer to loop over arrays with "of" since in general it's possible for "in" to include other properties besides array indices:
```suggestion
  for (const item of items) {
    if (fs.lstatSync(path.join(sourceDir, item)).isDirectory()) {
```